src/main/java/de/teamlapen/vampirism/entity/minion/management/PlayerMinionController.java

```diff
@@ -208,6 +209,8 @@ public MinionEntity<?> createMinionEntityAtPlayer(int id, @NotNull Player p) {
                 } else {
                     m.claimMinionSlot(id, this);
                     m.copyPosition(p);
+                    // load from the saved data, but do not check it out in case the entity is not loaded
+                    ((MinionEntity<MinionData>)m).handleLoadedMinionData(minions[id].data);
```

Comment:
@maxanier could this be a problem?

If the entity is added to the world it is checked out (and loaded from the minion data) otherwise the entity is removed.

Reply:
I will consider this later.